.ci_support/linux_64_.yaml

```diff
@@ -1,11 +1,12 @@
 cdt_name:
 - cos6
 channel_sources:
-- conda-forge,defaults
+- conda-forge
 channel_targets:
 - conda-forge main
 docker_image:
-- quay.io/condaforge/linux-anvil-comp7
-zip_keys:
-- - cdt_name
-  - docker_image
+- quay.io/condaforge/linux-anvil-cos7-x86_64
+ffmpeg:
+- '4.3'
```

Comment:
its kinda strange to me that this ffmpeg is appearing in the pinning given that it is only a runtime dependency.

I should ask other core members about this.